teleop_twist_keyboard.py

```diff
@@ -33,13 +33,17 @@
 
 import select
 import sys
-import termios
-import tty
-
-from geometry_msgs.msg import Twist
 
+import geometry_msgs.msg
 import rclpy
 
+if sys.platform == 'win32':
```

Comment:
@clalancette meta: I wonder if it'd make sense to move all console related functions to separate `win32` and non-`win32` modules, and then use this conditional once to `import`

Reply:
Yeah, I considered that, and also considered having a single module that had all of these `if sys.platform` statements embedded in it, so that `teleop_twist_keyboard.py` wouldn't have to worry about platform-specific differences.

In the end, I decided to do neither because this code is so short.  It's not super hard to follow, so I'd prefer to keep the simplicity of a single file unless this grows significantly more complex.